test/Elastic.Apm.Tests.MockApmServer/MockApmServer.cs

```diff
@@ -192,7 +192,11 @@ public async Task StopAsync()
 				{
 					services.AddMvc()
 						.AddApplicationPart(typeof(MockApmServer).Assembly)
+#if !NET5_0
 						.SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+#else
+						;
```

Comment:
```suggestion
						;
						services.AddControllers();
```

Looks like net5.0 needs this.

The `services.AddMvc() ...` is duplicated in `Startup.ConfigureServices()`. Suggest we remove it here